content/manual/os/s3.md

```diff
@@ -35,23 +35,25 @@ Finally you'll need to create an API key so you can configure your ship to have
 
 Click API in the bottom left of the DigitalOcean navigation bar on the left side of the webpage.
 
-Generate a new token, you'll need to copy the access key and secret. Don't navigate away from this page as the secret will only be shown when it is initially created.
+Generate a new `Spaces access key`. Note that this is distinct from a `Personal access token`. You'll need to copy the access key and secret. Don't navigate away from this page as the secret will only be shown when it is initially created.
```

Comment:
both of these options are on the same page and they can be easily confused
<img width="1244" alt="Screen Shot 2022-12-31 at 10 45 12 AM" src="https://user-images.githubusercontent.com/24712874/210148562-a05136f1-72c2-4771-867a-db64ede20c13.png">